tests/conftest.py

```diff
@@ -41,6 +124,75 @@ def staff_patron_all_values_dict():
     }
 
 
+@pytest.fixture()
+def student_database_record_with_all_values():
```

Comment:
Even though several of these are just used once, I think it's more readable to leave them in `conftest.py`